backend/aml/urls.py

```diff
@@ -26,31 +32,32 @@
 
 # Urls patterns
 urlpatterns = [
-    path('experiment/', include('experiment.urls')),
-    path('question/', include('question.urls')),
-    path('participant/', include('participant.urls')),
-    path('result/', include('result.urls')),
-    path('section/', include('section.urls')),
-    path('session/', include('session.urls')),
-    path('theme/', include('theme.urls')),
-    path('admin_interface/', include('admin_interface.urls')),
-    path('admin/', admin.site.urls),
-
+    path("experiment/", include("experiment.urls")),
+    path("question/", include("question.urls")),
+    path("participant/", include("participant.urls")),
+    path("result/", include("result.urls")),
+    path("section/", include("section.urls")),
+    path("session/", include("session.urls")),
+    path("theme/", include("theme.urls")),
+    path("admin_interface/", include("admin_interface.urls")),
+    path("admin/", admin.site.urls),
+    path("api/token/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
+    path("api/token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
```

Comment:
This is used by the experiment form react application to obtain a JWT token that is used to authenticate the user in Rest API requests